src/components/common/error/ErrorBar.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { NavLink, useHistory } from 'react-router-dom'
+import { URLS } from '../../../config'
+import { ReactComponent as ErrorInfo } from '../../../assets/icons/misc/errorInfo.svg'
+import { AppDetails } from '../../v2/appDetails/appDetails.type'
+
+function ErrorBar({ appDetails }: { appDetails: AppDetails }) {
+    let isImagePullBackOff = false
+    for (let index = 0; index < appDetails.resourceTree.nodes.length; index++) {
+        const node = appDetails.resourceTree.nodes[index]
+        if (node.info?.length) {
+            for (let index = 0; index < node.info.length; index++) {
+                const info = node.info[index]
+                if (info.value.toLowerCase() === 'errimagepull' || info.value.toLowerCase() === 'imagepullbackoff') {
+                    isImagePullBackOff = true
+                    break
+                }
+            }
+            if (isImagePullBackOff) break
+        }
+    }
+
+    return (
+        appDetails && isImagePullBackOff && !appDetails?.externalCi && (
```

Comment:
we should add this appDetails check on the top. if appDetails will be undefined then it anyways break the code